distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -1477,113 +1480,88 @@ public void testAccessGroupsGuest() throws Exception {
       getSecurityPolicy().waitForBasicAuthReady(url);
 
       // user with permissions gets results
-      response = getBasicRestResponseAsString(url, A_USER, USER_PASSWORD);
+      response = queryWithBasicAuth(id, TRANSFORMER_XML, A_USER, USER_PASSWORD);
       assertThat(response, containsString("Lady Liberty"));
 
       // user without permissions gets results
-      response = getBasicRestResponseAsString(url, B_USER, USER_PASSWORD);
+      response = queryWithBasicAuth(id, TRANSFORMER_XML, B_USER, USER_PASSWORD);
       assertThat(response, containsString("Lady Liberty"));
 
     } finally {
-      configureAuthZRealm(authZProperties, getAdminConfig());
+      if (authZProperties != null) {
+        configureAuthZRealm(authZProperties, getAdminConfig());
+      }
       if (metacardAttributeSecurityFilterProperties != null) {
         configureMetacardAttributeSecurityFiltering(
             metacardAttributeSecurityFilterProperties, getAdminConfig());
       }
       configureRestForGuest(SDK_SOAP_CONTEXT);
+      waitForSecurityHandlers(url);
       getSecurityPolicy().waitForGuestAuthReady(url);
-      // metacard will be deleted in @After
     }
   }
 
   @Test
   public void testAccessGroups() throws Exception {
-    List attr =
-        ImmutableList.of(
-            "security.access-groups=http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role");
-
-    Dictionary authZProperties = configureAuthZRealm(attr, getAdminConfig());
-
+    Dictionary authZProperties = null;
     Dictionary metacardAttributeSecurityFilterProperties = null;
 
-    String url = null;
+    String testData =
```

Comment:
✏️ Could use this `getFileContent` instead. https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L815